README.md

```diff
@@ -106,21 +117,21 @@ For more information on cases you should handle during layout, please refer to [
 
 FluentDarkModeKit will notify views or view controllers in the current window when the theme changes by calling the following delegate method. 
 
-Swift
 ```swift
 protocol DMTraitEnvironment: NSObjectProtocol {
     func dmTraitCollectionDidChange(_ previousTraitCollection: DMTraitCollection?)
 }
 ```
 
-Objective-C
-```objc
-@protocol DMTraitEnvironment <NSObject>
+## Migration
 
-- (void)dmTraitCollectionDidChange:(nullable DMTraitCollection *)previousTraitCollection;
+If you are using a pre 0.5.2 version and wants to migrate to the new version, here are the changes that you should pay attention to:
+
+1. On iOS 13, the latest version of FluentDarkModeKit uses iOS 13's API for dynamic color and theme change.

```

Comment:
I think we should explain in detail what the differences are. For example, `DarkModeManager.register(with: UIApplication.shared)` is required for FluentDarkModeKit to actually work.

Also, do you think we need to add an explicit warning on the top of README and provide the link to this section?

Reply:
as for the required method, I think I've put an anchor in Migration that links to the "How to Use FluentDarkModeKit" section which covers the set up methods.

probably we don't need an explicit warning, as project with older version of FluentDarkModeKit will not compile. and a big warning can be intrusive and unfriendly